cirq/study/visualize_test.py

```diff
@@ -38,3 +38,40 @@ def test_plot_state_histogram():
     expected_values = [0., 0., 0., 5.]
 
     np.testing.assert_equal(values_plotted, expected_values)
+
+
+def test_plot_state_histogram_multi_1():
```

Comment:
Add the line `pl.switch_backend('PDF')` to ensure plots aren't displayed while running tests.